src/app/core/learning-object-module/learning-object/learning-object.routes.ts

```diff
@@ -122,6 +137,28 @@ export const LEARNING_OBJECT_ROUTES = {
     DELETE_LEARNING_OBJECT(learningObjectId: string) {
         return `${environment.apiURL}/learning-objects/${encodeURIComponent(learningObjectId)}`;
     },
+
+    POST_MAPPING(username: string, learningObjectId: string, outcomeId: string) {
+        return `${environment.apiURL}/users/${encodeURIComponent(
+            username,
+        )}/learning-objects/${encodeURIComponent(
+            learningObjectId,
+        )}/outcomes/${encodeURIComponent(outcomeId)}/mappings`;
+    },
+    DELETE_MAPPING(
+          username: string,
+          learningObjectId: string,
+          outcomeId: string,
+          mappingsId: string,
+      ) {
+          return `${environment.apiURL}/users/${encodeURIComponent(
+              username,
+          )}/learning-objects/${encodeURIComponent(
+              learningObjectId,
+          )}/outcomes/${encodeURIComponent(outcomeId)}/mappings/${encodeURIComponent(
+              mappingsId,
+          )}`;
+      },
```

Comment:
Are these the paths that are used? I thought most of the routes we got rid of `/users/${username}...` Are these from standard guidelines?